src/components/ViolationForm.js

```diff
@@ -0,0 +1,250 @@
+        </option>
+      );
+    });
+  };
+  const getMunicipalities = (selectedElectionRegion) => {
+    console.log('here');
+
+    const filteredRegions = electionRegions.filter(
+      (electionRegion) => electionRegion.code == selectedElectionRegion
+    );
+    const municipalities = [];
+    filteredRegions[0].municipalities.forEach((municipality) => {
+      municipalities.push(municipality);
+    });
+
+    return municipalities.map((municipality) => {
+      return (
+        <option key={municipality.code} value={municipality.code}>
+          {municipality.name}
+        </option>
+      );
+    });
+  };
+  const createTownOptions = () => {
+    console.log('in create town');
+    return towns.map((town) => {
+      return (
+        <option id={town.code} key={town.code} value={town.code}>
+          {town.name}
+        </option>
+      );
+    });
+  };
+  const onSubmit = (data) => {
+    const body = {
+      description: data.description,
+      town: 68134,
+    };
+    console.log(JSON.stringify(body));
+    axios
+      .post(`${api_endpoint}/violations`, body, {
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      })
+      .then((res) => console.log(res.data))
+      .catch((err) => console.log(err));
+  };
+
+  return (
+    <CommentFormStyle onSubmit={handleSubmit(onSubmit)}>
+      <div className="form-control">
+        <label>Секция в:</label>
+        <div className="radio_button_1">
+          <input
+            type="radio"
+            id="fieldBg"
+            {...register('fieldBg', { required: false })}
+          />
+          <label className="radioLabel">България</label>
+          <input
+            type="radio"
+            id="fieldForeign"
+            {...register('fieldForeign', { required: false })}
+          />
+          <label className="radioLabel">Чужбина</label>
+        </div>
+      </div>
+      <div>
+        <label>МИР</label>
+      </div>
+      <div>
+        <select
+          className="form-control"
+          onChange={(e) => setSelectedElectionRegion(e.target.value)}
+        >
+          <option value="choose" disabled selected="selected">
+            -- МИР --
+          </option>
+          {getElectionRegions()}
+        </select>
+      </div>
+      <div>
+        <label>Община</label>
+      </div>
+      <div>
+        <select
+          className="form-control"
+          onChange={(e) => setSelectedMunicipality(e.target.value)}
+        >
+          {selectedElectionRegion != '' ? (
+            getMunicipalities(selectedElectionRegion)
+          ) : (
+            <option value="choose" disabled selected="selected">
+              -- Община --
+            </option>
+          )}
+        </select>
+      </div>
+      <div>
+        <label>Град/село</label>
+      </div>
+      <div>
+        <select className="form-control">
+          {towns.length != 0 ? (
+            createTownOptions()
+          ) : (
```

Comment:
If the town has city regions, we should also add dynamically a drop down for city regions.

Exactly like how the mobile app behaves.